packages/shared/__tests__/fixture/auth.ts

```diff
@@ -157,7 +157,7 @@ export const errorRegistrationMockData: Partial<InitializationData> = {
             },
           },
         ],
-        meta: { label: { id: 1070002, text: 'Full name', type: 'info' } },
+        meta: { label: { id: 1070002, text: 'Name', type: 'info' } },
```

Comment:
Actually dont think these need changing, will need to manually verify if the error states work now as in heimdall name would be full name